docs/quickstart/azure.md

```diff
@@ -0,0 +1,113 @@
+```
+<img src="../images/quickstart/azure/azure7.png">
+
+Exit vim and run the following command to start Coder as a system level service:
+
+```sh
+ sudo systemctl enable --now coder
+```
+
+The following command will get you information about the Coder launch service
+
+```sh
+ journalctl -u coder.service -b
+```
+
+This will return a series of logs related to running Coder as a system service. Embedded in the logs is the Coder Access URL.
+
+Copy the URL and run the following command to create the first user, either on your local machine or in the instance terminal. 
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts. Be sure to save use email and password as these are your admin username and password.
+
+You can now access Coder on your local machine with the relevant  `***.try.coder.app` URL and logging in with the username and password.
+
+## Creating and Uploading Your First Template
+
+First, run `coder template init` to create your first template. You’ll be given a list of possible templates to use. This tutorial will show you how to set up your Coder instance to create a Linux based machine on Azure. 
+
+<img src="../images/quickstart/azure/azure9.png">
+
+Press `enter` to select `Develop in Linux on Azure` template. This will return the following: 
+
+<img src="../images/quickstart/azure/azure10.png">
+
+To get started using the Azure template, install the Azure CLI by following the instructions [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt). Run `az login` and follow the instructions to configure the Azure command line. 
+
+Coder is running as a system service, which creates the system user `coder` for handling processes. The Coder user will require access to the Azure credentials to initialize the template.
+
+Run the following commands to copy the Azure credentials and give the `coder` user access to them:
+
+```sh
+sudo cp -r ~/.azure /home/coder/.azure 
+sudo chown -R coder:coder /home/coder/.azure/
+```
+
+Navigate to the `./azure-linux` folder where you created your template and run the following command to put the template on your Coder instance. 
+
+```sh
+coder templates create
+```
+
+Congrats! You can now navigate to your Coder dashboard and use this Linux on Azure template to create a new workspace!
+
+## Troubleshooting
+
+If you’re not sure if Coder is running correctly as a system service, use the command `systemctl status coder` to show the status of the Coder process.
+
+To stop running Coder as a system service, run the following command: `systemctl stop coder`. 
+
+When you run ` journalctl -u coder.service -b`, there’s a chance that it may not show any logs, preventing you from seeing the url for the running Coder instance. Use the command `systemctl stop coder` to stop the Coder system service, and run the command to start Coder as a system service again. You can also use the command `systemctl restart coder` to quickly restart the service. 
```

Comment:
<same as above>

How about we move Troubleshooting to another PR so we can merge the meat of this one?